.github/workflows/flathub-data-checker.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Check if external data changed'
```

Comment:
I don't think you want to keep anything from .github folder, or target new-pr branch instead

Reply:
Right. I will delete that